swervelib/Mk4ModuleConfiguration.java

```diff
@@ -2,6 +2,8 @@
 
 import java.util.Objects;
 
+import frc.team670.mustanglib.swervelib.redux.AbsoluteEncoderType;
```

Comment:
why is this part of this change?

and also: it's weird that AbsoluteEncoderType is under a redux package - having it there already implies a type